CHANGES.md

```diff
@@ -68,6 +68,7 @@
 ## New Features / Improvements
 
 * X feature added (Java/Python) ([#X](https://github.com/apache/beam/issues/X)).
+* The Python SDK now type checks `collections.abc.Collections` types properly. Some type hints that were erroneously allowed by the SDK may now be failed. ([#29272](https://github.com/apache/beam/pull/29272))
```

Comment:
if you have actionable suggestions how to fix that, it would be great. See example: https://github.com/apache/beam/blob/master/CHANGES.md#breaking-changes-9

Reply:
It's kind of tricky in that this is technically a bug fix, all of the internal fixes I had to put together were something along the lines of "correct code, incorrect hint for the code" with Collection hints when Iterables were provided as a common example.